.github/workflows/ci-helm.yml

```diff
@@ -0,0 +1,40 @@
+name: Helm CI
+on:
+  push:
+    branches: [main]
+  pull_request:
+    paths:
+      - 'docs/deployment/azure-scripts'
+      - .github/workflows/ci-helm.yml
+
+env:
+  HELM_EXPERIMENTAL_OCI: 1 #enable OCI support
+  HELM_VERSION_TO_INSTALL: 3.10.2 # version of HEL to install
+  GCR_IMAGE: ghcr.io/microsoft/showwhy/
+
+jobs:
+  build:
+    name: publish gcr
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: checkout repo
+      
+      - name: install helm
+        uses: Azure/setup-helm@v1
+        with:
+          # Version of helm
+          version: ${{ env.HELM_VERSION_TO_INSTALL }} # default is latest
+      
+      - name: login to acr using helm
```

Comment:
Maybe we should put some conditions to only login when the ref is the main branch?